src/lib/pickles/tests/test_ro.ml

```diff
@@ -0,0 +1,20 @@
+(* initially written to see the consistency between updates of digestif *)
+let test_bits_random_oracle_consistency_check () =
+  let s = "BitsRandomOracle" in
+  let exp_output_str =
+    "01000001101110001111111100011000011000100010100011010010110110010011101101101000001110001110100101010100001000001110101110111010"
+  in
+  let exp_output =
+    List.map (fun i -> i = '1') (List.of_seq (String.to_seq exp_output_str))
```

Comment:
```suggestion
    List.map (fun i -> i = '1') (String.to_list exp_output_str)
```
Or, for OCaml golfer (see [here](https://www.vimgolf.com/) for the reference to golf) who wants to spare iterations
```suggestion
    let l = ref [] in 
    for i = String.length exp_output_str - 1 downto 0 do
        l := exp_output_str.(i) = '1' :: !l
    done;
    !l
```
:-)

Reply:
Oh, it seems Core_kernel implements the to_list function in the String module. That's nice! Gonna start liking Core...

Reply:
For a test, I prefer to stay functional :)

Reply:
Then rewrite the for loop as 
```
let rec loop acc i = if i < 0 then acc else loop (exp_output_str.(i) = '1' :: acc) (i - 1) in 
loop [] (String.length exp_output_str - 1)
```
;-)

In any case, it's high-level nitpicking since it's just a single test.